app/comments/get.controller.js

```diff
@@ -0,0 +1,28 @@
+const { logger } = require('../../common/logging/logger')
+const { getSentencePlanComments } = require('../../common/data/sentencePlanningApi')
+const { getCommentText } = require('../../common/utils/getCommentText')
+
+const getComments = async (
+  { path, errors, errorSummary, body, renderInfo, params: { planId }, session: { 'x-auth-token': token } },
+  res
+) => {
+  const renderDetails = renderInfo || {}
+
+  renderDetails.nexturl = path.substring(0, path.lastIndexOf('/'))
+  renderDetails.backurl = `${path.substring(0, path.lastIndexOf('/'))}/decisions`
+
+  if (body.comments) {
+    renderDetails.comments = body.comments
+  } else {
+    try {
+      const comments = await getSentencePlanComments(planId, token)
+      renderDetails.comments = getCommentText(comments, 'THEIR_SUMMARY')
```

Comment:
'THEIR_SUMMARY' - I think I'm using this text const's all over the place. I think we create a shared constants file and reference. Gives us a degree of security for any future uses or changes. Worth raising a ticket?

Reply:
Created as https://dsdmoj.atlassian.net/browse/SP-482